includes/admin.php

```diff
@@ -372,6 +387,28 @@ function handle_ajax_delete_transients() {
 	);
 
 	die();
+	}
+
+/**
+ * Handles the AJAX request for disabling webhooks.
+ *
+ * @return void
+ */
+function handle_ajax_disable_webhooks() {
+
+	// Permissions and security checks.
+	check_the_permissions();
+	check_the_nonce( $_POST['nonce'], 'safety-net-disable-webhooks' ); // phpcs:ignore WordPress.Security.NonceVerification
+
+	// Checks passed. Disable the webhooks.
+	disable_webhooks();
+
+	echo wp_json_encode(
+		array(
+			'success' => true,
+			'message' => esc_html__( 'Webhooks have been disabled.' ),
```

Comment:
This is weird, and I don't know why it happens, but if I go to `Tools` and click Disable Webhooks button, the JSON response comes back with a `0` at the end, causing the page to keep spinning forever.

<img width="933" alt="Screenshot 2025-02-20 at 10 44 49 AM" src="https://github.com/user-attachments/assets/f67131fd-ac41-4b14-a78b-39d12ab71701" />

I cannot replicate the issue with other options like Scrub Options, Deactivate Plugins or Delete Transients.

Could it be because I manually uploaded the plugin to via SFTP to the `mu-plugins` directory? Still, my code doesn't have that `0` anywhere, so I'm not sure what's causing it. Feel free to ignore if you cannot reproduce.

Reply:
Turns out this is caused by the `die(0)` in the core `admin-ajax.php` file. It explains the `die()` statement in the other functions in here, so I added it to this one in ebf5b7e.

I had this behavior crop up only after I deployed it and thought it was a one-off. Thanks for the catch!